Objects/Converters/ConverterAutocadCivil/ConverterAutocadCivilShared/ConverterAutocadCivil.cs

```diff
@@ -102,167 +104,189 @@ public Base ConvertToSpeckle(object @object)
-            @base = CorridorToSpeckle(o);
-            break;
-          case CivilDB.FeatureLine o:
-            @base = FeatureLineToSpeckle(o);
-            break;
-          case CivilDB.Structure o:
-            @base = StructureToSpeckle(o);
-            break;
-          case CivilDB.Pipe o:
-            @base = PipeToSpeckle(o);
-            break;
-          case CivilDB.PressurePipe o:
-            @base = PipeToSpeckle(o);
-            break;
-          case CivilDB.Profile o:
-            @base = ProfileToSpeckle(o);
-            break;
-          case CivilDB.TinSurface o:
-            @base = SurfaceToSpeckle(o);
-            break;
-
-#elif ADVANCESTEEL
-
-          default:
-            try
-            {
-              @base = ConvertASToSpeckle(obj, reportObj, notes);
-            }
-            catch (Exception ex)
-            {
-              //Update report because AS object type
-              Report.UpdateReportObject(reportObj);
-              throw;
-            }
-
-            break;
+            case CivilDB.Alignment o:
+              @base = AlignmentToSpeckle(o);
+              break;
+            case CivilDB.Corridor o:
+              @base = CorridorToSpeckle(o);
+              break;
+            case CivilDB.FeatureLine o:
+              @base = FeatureLineToSpeckle(o);
+              break;
+            case CivilDB.Structure o:
+              @base = StructureToSpeckle(o);
+              break;
+            case CivilDB.Pipe o:
+              @base = PipeToSpeckle(o);
+              break;
+            case CivilDB.PressurePipe o:
+              @base = PipeToSpeckle(o);
+              break;
+            case CivilDB.Profile o:
+              @base = ProfileToSpeckle(o);
+              break;
+            case CivilDB.TinSurface o:
+              @base = SurfaceToSpeckle(o);
+              break;
 #endif
-        }
-        break;
-      case Acad.Geometry.Point3d o:
-        @base = PointToSpeckle(o);
-        break;
-      case Acad.Geometry.Vector3d o:
-        @base = VectorToSpeckle(o);
-        break;
-      case Acad.Geometry.Line3d o:
-        @base = LineToSpeckle(o);
-        break;
-      case Acad.Geometry.LineSegment3d o:
-        @base = LineToSpeckle(o);
-        break;
-      case Acad.Geometry.CircularArc3d o:
-        @base = ArcToSpeckle(o);
-        break;
-      case Acad.Geometry.Plane o:
-        @base = PlaneToSpeckle(o);
-        break;
-      case Acad.Geometry.Curve3d o:
-        @base = CurveToSpeckle(o) as Base;
-        break;
-      default:
-        if (reportObj != null)
-        {
-          reportObj.Update(status: ApplicationObject.State.Skipped, logItem: $"{@object.GetType()} type not supported");
-          Report.UpdateReportObject(reportObj);
-        }
-        return @base;
+            default:
+#if ADVANCESTEEL
+              try
+              {
+                @base = ConvertASToSpeckle(obj, reportObj, notes);
+              }
+              catch (Exception)
```

Comment:
Potentially missing `IsFatal`, not sure we want to try and update the report when `OutOfMemory` for example.